android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/CheckBoxFactory.java

```diff
@@ -161,7 +161,7 @@ private void addRequiredValidator(LinearLayout rootLayout, JSONObject jsonObject
         }
     }
 
-    private ArrayList<View> addCheckBoxOptionsElements(JSONObject jsonObject, Context context, Boolean readOnly,
+    private ArrayList<View> addCheckBoxOptionsElements(JSONObject jsonObject, final Context context, Boolean readOnly,
```

Comment:
Since this is a private method, we can refactor it to use `WidgetArgs` https://github.com/opensrp/opensrp-client-native-form/blob/c9c6c7e1ecebc2c9a8d1f83394237072d4b805cf/android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/RepeatingGroupFactory.java#L87 and shorten the number of params it needs to be passed.

Reply:
Done, make use of WidgetArgs class to reduce params passed to the method.